### ISSUE TITLE
[MAS2.4.7][Keyboard Navigation- Live Chat Dialog] Focus is not visible on "Attachment" button.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1960]
**Actual Result:**
The keyboard focus is not properly visible on "Attachment" button available at "Live Chat" dialog and "Presentation Mode" screen.

**Expected Result:**
The focus should be visible properly on "Attachment" button as the focus should visible properly on each and every control of the application. The rectangular focus should be appears just like "Send" button.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to any already created bot and select any bot..
4. "Live Chat" screen will appear.
5. Now, navigate to "Attachment" button.
6. Observe the issue.

**User Impact:**
When a component can receive keyboard focus and the focus is not visible to the user. User will not able to activate the component using the keyboard. 

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B3b98b04f-4fab-4d3e-81af-a0906a34849c%7D](url)

**Attachment:**
[Focus is not visible on Attachment button.zip](https://github.com/microsoft/BotFramework-Emulator/files/3797887/Focus.is.not.visible.on.Attachment.button.zip)

**Comment by tonyanziano** [comment]
This has been fixed and is ready to be verified in the latest nightly version:

![image](https://user-images.githubusercontent.com/3452012/70383840-dcfad100-1929-11ea-82a7-7338c7d66f21.png)

**Comment by ShikhaMishra11** [comment]
We have checked in the latest nightly, the issue is fixed in "Live Chat Dialog" but now we don't have "Presentation Mode" button. I guess this is the design of the application. So, we are closing the bug.

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0.94518
